app/src/main/java/com/example/nguyenthanhtungh/myapplication/ui/adapter/HeroAdapter.java

```diff
@@ -14,37 +13,40 @@
 import java.util.ArrayList;
 
 public class HeroAdapter extends RecyclerView.Adapter<HeroAdapter.HeroHolder> {
-    private ArrayList<Hero> heroArrayList;
+    private ArrayList<Hero> mListHeros;
 
     public HeroAdapter(ArrayList<Hero> heroArrayList) {
-        this.heroArrayList = heroArrayList;
+        this.mListHeros = heroArrayList;
     }
 
     @NonNull
     @Override
     public HeroHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
         View view = LayoutInflater.from(viewGroup.getContext())
-                .inflate(R.layout.adapter_hero_items,viewGroup,false);
+                .inflate(R.layout.adapter_hero_items, viewGroup, false);
         return new HeroHolder(view);
     }
 
     @Override
     public void onBindViewHolder(@NonNull HeroHolder heroHolder, int i) {
-        Hero hero = heroArrayList.get(i);
-        heroHolder.imageHero.setImageResource(hero.getmImageId());
+        Hero hero = mListHeros.get(i);
+        heroHolder.mImageHero.setImageResource(hero.getImageId());
     }
 
     @Override
     public int getItemCount() {
-        return heroArrayList.size();
+        if (mListHeros == null) {
+            return 0;
+        }
+        return mListHeros.size();
```

Comment:
dùng toán tử điều kiện này đẹp hơn chú 
mListHeros !=null : mListHeros.size() ? 0;